Product/pom.xml

```diff
@@ -1132,6 +1133,11 @@
         <version>1.4.1</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
```

Comment:
I am not changing by modifying pom.xml text... i am changing by simply removing and adding through dependencies... please look at the screen shot i had sent you twice since morning.
If I add version in original code in line 664... it gives error... you can try yourself